packages/core/plugins/package.json

```diff
@@ -18,7 +18,11 @@
     "access": "public"
   },
   "devDependencies": {
+    "chart.js": "^4.2.1",
     "parcel": "^2.8.2",
     "typescript": "^4.9.4"
+  },
+  "peerDependencies": {
+    "chart.js": "^4.2.1"
```

Comment:
We want the customer to be responsible for installing chart.js along with our library themselves?

Reply:
No, the customer won't install this `plugins` package, the `plugins` package will be used by the components and the components will have their own `chart.js` dependency

Reply:
Interesting. OK